apps/phpcs.xml

```diff
@@ -25,6 +25,18 @@
 	<!-- Show sniff codes in all reports. -->
 	<arg value="ps"/>
 
+	<!-- Newspack and ETK don't use exactly the same code standard, so we need to
+		ignore style rules not tracked upstream. (We should not ignore rules
+		which catch actual errors.)
+	-->
+	<rule ref="VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable" >
```

Comment:
@jeffersonrabb this rule is tracked in our PHPCS but not in Newspack -- maybe worth adding there? I'm not sure where you pull your rules from, but we've just started pulling Jetpack's config for consistency.

Reply:
Good idea - added in https://github.com/Automattic/newspack-blocks/pull/702

Reply:
cool!